testsuite/integration/basic/src/test/java/org/jboss/as/test/integration/ejb/validation/EjbBeanValidationTestCase.java

```diff
@@ -60,10 +65,9 @@
         return ShrinkWrap.create(WebArchive.class, "ejbvalidation.war")
                 .addPackage(HttpRequest.class.getPackage())
                 .addClasses(
-                        EjbBeanValidationTestCase.class,
-                        TestApplication.class,
-                        TestResource.class
-                );
+                        EjbBeanValidationTestCase.class, TestApplication.class, TestResource.class, EchoResourceImpl.class, EchoResource.class,
+                        DummySubclass.class, DummyAbstractClass.class, DummyClass.class, DummyFlagImpl.class, DummyFlag.class)
+                .addAsWebInfResource("web.xml");
```

Comment:
`addAsWebInfResource("web.xml")` caused deployment failure, since there is no such file. This test case does not need web.xml so this line can be deleted.

Reply:
fixed